Buildings/Controls/OBC/Utilities/Validation/PIDWithInputGains.mo

```diff
@@ -0,0 +1,100 @@
+within Buildings.Controls.OBC.Utilities.Validation;
+model PIDWithInputGains "Test model for PIDWithInputGains"
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Pulse pulse(
+    period=0.25)
+    "Setpoint"
+    annotation (Placement(transformation(extent={{-30,14},{-10,34}})));
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant const(
+    k=0.5)
+    "Measurement data"
```

Comment:
These are not measurement data. Please review and update accordingly.

Reply:
Can you please let me if "measurement value" is a good word?